### ISSUE TITLE
Language update throws PHP Strict standards

### ISSUE THREAD

**Issue by martijn94** [wp-cli/wp-cli#2915]
The command "wp core language update" throws 2 stricts standards errors to me, this is with WP_DEBUG enabled.

```
vagrant@martijn:/vagrant/benegas$ wp core language update
PHP Strict standards:  Only variables should be passed by reference in phar:///usr/local/bin/wp/php/WP_CLI/CommandWithTranslation.php on line 169
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/wp:0
PHP   2. include() /usr/local/bin/wp:4
PHP   3. include() phar:///usr/local/bin/wp/php/boot-phar.php:5
PHP   4. WP_CLI\Runner->start() phar:///usr/local/bin/wp/php/wp-cli.php:21
PHP   5. WP_CLI\Runner->_run_command() phar:///usr/local/bin/wp/php/WP_CLI/Runner.php:757
PHP   6. WP_CLI\Runner->run_command() phar:///usr/local/bin/wp/php/WP_CLI/Runner.php:334
PHP   7. WP_CLI\Dispatcher\Subcommand->invoke() phar:///usr/local/bin/wp/php/WP_CLI/Runner.php:327
PHP   8. call_user_func:{phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/Subcommand.php:351}() phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/Subcommand.php:351
PHP   9. WP_CLI\Dispatcher\CommandFactory::WP_CLI\Dispatcher\{closure}() phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/Subcommand.php:351
PHP  10. call_user_func:{phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/CommandFactory.php:67}() phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/CommandFactory.php:67
PHP  11. WP_CLI\CommandWithTranslation->update() phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/CommandFactory.php:67
PHP Strict standards:  Only variables should be passed by reference in phar:///usr/local/bin/wp/php/WP_CLI/CommandWithTranslation.php on line 181
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/wp:0
PHP   2. include() /usr/local/bin/wp:4
PHP   3. include() phar:///usr/local/bin/wp/php/boot-phar.php:5
PHP   4. WP_CLI\Runner->start() phar:///usr/local/bin/wp/php/wp-cli.php:21
PHP   5. WP_CLI\Runner->_run_command() phar:///usr/local/bin/wp/php/WP_CLI/Runner.php:757
PHP   6. WP_CLI\Runner->run_command() phar:///usr/local/bin/wp/php/WP_CLI/Runner.php:334
PHP   7. WP_CLI\Dispatcher\Subcommand->invoke() phar:///usr/local/bin/wp/php/WP_CLI/Runner.php:327
PHP   8. call_user_func:{phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/Subcommand.php:351}() phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/Subcommand.php:351
PHP   9. WP_CLI\Dispatcher\CommandFactory::WP_CLI\Dispatcher\{closure}() phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/Subcommand.php:351
PHP  10. call_user_func:{phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/CommandFactory.php:67}() phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/CommandFactory.php:67
PHP  11. WP_CLI\CommandWithTranslation->update() phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/CommandFactory.php:67
Updating 'Dutch' translation for Regenerate Thumbnails 2.2.6...
Downloading translation from https://downloads.wordpress.org/translation/plugin/regenerate-thumbnails/2.2.6/nl_NL.zip...
Translation updated successfully.
Success: Updated 1/1 translations.
```
My WP --info output:
```
PHP binary:	/usr/bin/php5
PHP version:	5.6.15-1+deb.sury.org~trusty+1
php.ini used:	/etc/php5/cli/php.ini
WP-CLI root dir:	phar://wp-cli.phar
WP-CLI packages dir:	/home/vagrant/.wp-cli/packages/
WP-CLI global config:	
WP-CLI project config:	
WP-CLI version:	0.23.1
```
If there is any more debug info needed, please let me know!

**Comment by martijn94** [comment]
Thanks!

**Comment by grappler** [comment]
This issue does not seemed to be fixed as I am getting this error in `v0.24.1`. The PR did not touch the line 169 where the error was coming from. The code being `$plugin_data = array_shift( get_plugins( '/' . $update->slug ) );`

Today the code in line 195 https://github.com/wp-cli/wp-cli/blob/master/php/WP_CLI/CommandWithTranslation.php#L195

```
PHP binary:    	/usr/local/php/bin/php_cli
PHP version:   	7.0.9
php.ini used:  	/etc/php/php.ini
WP-CLI root dir:       	phar://wp-cli.phar
WP-CLI packages dir:
WP-CLI global config:
WP-CLI project config:
WP-CLI version:	0.24.1
```
